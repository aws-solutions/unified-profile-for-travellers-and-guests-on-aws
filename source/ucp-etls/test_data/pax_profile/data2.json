{
  "modelVersion": "1.0",
  "id": "2296805072",
  "lastUpdatedOn": "2022-06-17T18:47:51.048086961Z",
  "createdOn": "2022-04-17T18:47:51.048086961Z",
  "lastUpdatedBy": "Delfina Farrell",
  "createdBy": "Raoul Mertz",
  "emails": [
    {
      "type": "",
      "address": "toreyschmitt@schmeler.name",
      "primary": false
    }
  ],
  "phones": [
    {
      "type": "",
      "number": "(984)115-3468",
      "primary": true,
      "countryCode": 26
    },
    {
      "type": "home",
      "number": "490-735-1883",
      "primary": true,
      "countryCode": 96
    }
  ],
  "addresses": [
    {
      "type": "shipping",
      "line1": "537 Bypass view",
      "line2": "more content line 2",
      "line3": "more content line 3",
      "line4": "more content line 4",
      "city": "Hialeah",
      "stateProvince": {
        "code": "ND",
        "name": ""
      },
      "postalCode": "24477",
      "country": {
        "code": "PM",
        "name": ""
      },
      "primary": false
    },
    {
      "type": "shipping",
      "line1": "87692 West Cliffs chester",
      "line2": "more content line 2",
      "line3": "more content line 3",
      "line4": "more content line 4",
      "city": "Mesa",
      "stateProvince": {
        "code": "TX",
        "name": ""
      },
      "postalCode": "78682",
      "country": {
        "code": "RS",
        "name": ""
      },
      "primary": false
    }
  ],
  "honorific": "Dr",
  "firstName": "Daija",
  "middleName": "Jacquelyn",
  "lastName": "Rowe",
  "gender": "other",
  "pronoun": "he",
  "dateOfBirth": "1909-12-02",
  "language": {
    "code": "zh",
    "name": ""
  },
  "nationality": {
    "code": "IQ",
    "name": ""
  },
  "jobTitle": "Officer",
  "parentCompany": "The Bridgespan Group",
  "loyaltyPrograms": [
    {
      "id": "1760240805",
      "programName": "elite",
      "miles": 798571.0,
      "milesToNextLevel": 865.0,
      "level": "platinium",
      "joined": "2022-04-17T18:47:51.048086961Z"
    }
  ],
  "identityProofs": null
}