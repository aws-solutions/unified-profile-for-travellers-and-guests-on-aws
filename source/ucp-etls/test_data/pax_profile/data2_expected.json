{"air_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "id": "1598507532", "last_updated": "2023-03-18T07:37:30.056203561Z", "created_on": "2023-02-18T07:37:30.056203561Z", "last_updated_by": "Minerva Kulas", "created_by": "German Kertzmann", "honorific": "Sir", "first_name": "Fidel", "middle_name": "Lilliana", "last_name": "Wolff", "gender": "male", "pronoun": "", "date_of_birth": "1919-12-24", "job_title": "Developer", "company": "Thomson Reuters", "nationality_code": "NZ", "nationality_name": "", "language_code": "sc", "language_name": "", "email_business": "roselynwalker@strosin.net", "email": "", "email_type": "business", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "1598507532", "last_updated_partition": "2023/03/18/07/37/30/056203561Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": ""}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2023-03-18T07:37:30.056203561Z", "last_updated_by": "Minerva Kulas", "address": "roselynwalker@strosin.net", "type": "business", "traveller_id": "1598507532", "last_updated_partition": "2023/03/18/07/37/30/056203561Z"}], "common_phone_recs": [], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2023-03-18T07:37:30.056203561Z", "last_updated_by": "Minerva Kulas", "id": "5357859349", "program_name": "ascend", "miles": 489704.0, "miles_to_next_level": 8557.0, "level": "platinium", "joined": "2023-02-18T07:37:30.056203561Z", "traveller_id": "1598507532", "last_updated_partition": "2023/03/18/07/37/30/056203561Z"}]}